ldai/tracker.go

```diff
@@ -43,11 +45,13 @@ func (t TokenUsage) Set() bool {
 type Metrics struct {
 	// Latency is the latency of the request.
 	Latency time.Duration
+	// TimeToFirstToken is the time to the first token of the streamed response.
+	TimeToFirstToken time.Duration
 }
 
 // Set returns true if the latency is non-zero.
 func (m Metrics) Set() bool {
-	return m.Latency != 0
+	return m.Latency != 0 || m.TimeToFirstToken != 0
```

Comment:
Given the refactoring below, it doesn't appear like this method is still in use anywhere.

We could remove this method, and then this would be a breaking change (which is fine since we're pre-v1.

Or if we really don't want to remove it, we should at least update the documentation to match the current behavior.

Reply:
i only kept it for fear of breaking change. i'll remove it!